tokens/connect/subwallet.md

```diff
@@ -0,0 +1,149 @@
+
+Next, you'll be able to provide the relevant seed phrase, private key, JSON file, or QR code, and you can begin using your new account right away.
+
+## Connect SubWallet to Moonbeam {: #connect-subwallet-to-moonbeam }
+
+To configure SubWallet for Moonbeam, select the hamburger menu from the top left corner, which will take you to the settings page.
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-9.png)
+
+From the settings menu, click **Manage networks**.
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-10.png)
+
+To add Moonbeam, you can:
+
+1. Search for "Moon" to view all Moonbeam-based networks, or search for a specific network
+2. Toggle the switch to connect to the network
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-11.png)
+
+If you're trying to connect to a [local Moonbeam development node](/builders/get-started/networks/moonbeam-dev){target=_blank}, you can click the **+** icon in the top right corner and enter in the [network configurations](/builders/get-started/quick-start/#network-configurations){target=_blank}.
+
+By default, all balances are hidden in SubWallet, but if you press the eye icon, you can toggle balance visibility.
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-12.png)
+
+## Interact with the Network {: #interact-with-the-network }
+
+Once you've [connected SubWallet](#connect-subwallet-to-moonbeam) to any Moonbeam-based network, you can start using your wallet by:
+
+- Sending a token transfer to another address
+- Adding ERC-20s to Metamask and interacting with them
+- Adding ERC-721s to Metamask and interacting with them
+
+### Send a Transaction {: #send-a-transaction }
+
+To get started with a simple token transfer to another address on Moonbeam, you can click the send icon.
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-13.png)
+
+Next, you can take the following steps:
+
+1. Specify the asset to send and the destination chain (in this case, the same chain that you're sending from)
+2. Enter the destination address
+3. Enter the amount of tokens to send
+4. Look over the transaction details, then press **Transfer**
+
+![The transfer screen on the SubWallet browser extension, where you can enter in the transaction details.](/images/tokens/connect/subwallet/subwallet-14.png)
+
+On the next screen, you'll be able to review the transaction details and submit the transaction. If the transaction details look good, you can click **Approve** to send the transaction.
+
+![The transfer confirmation screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-15.png)
+
+After you send the transaction, you'll be able to review the transaction details.
```

Comment:
It would be great to have a sentence or two wrap here perhaps directing people to, for more information about Subwallet, check out their user guide here: https://docs.subwallet.app/main/